resources/styles/components/learning-guide/guide-key-mixin.less

```diff
@@ -4,6 +4,7 @@
   .item {
     .flex-display();
     .align-items(center);
+    -ms-flex-align: center;
```

Comment:
Yes, that makes sense.  The styles I added are the old flexbox types and are only used by IE 10.  Would be nice to yank them out eventually.